python/tank/bootstrap/baked_configuration.py

```diff
@@ -64,6 +65,15 @@ def __init__(
         super(BakedConfiguration, self).__init__(path, descriptor)
         self._path = path
         self._sg_connection = sg
+
+        # Ensure that project_id, plugin_id and pipeline_config_id are not binary.
```

Comment:
Why would they be? Is it because they are coming from a parsed file? If so, we should fix the file parser instead to always return text.

Reply:
✅